plugin/helpers_config.go

```diff
@@ -24,7 +28,17 @@ func (p *HelpersImpl) CheckRequiredServerConfiguration(req *model.Config) (bool,
 	}
 
 	mergedCfg := mc.(model.Config)
-	if mergedCfg.ToJson() != cfg.ToJson() {
+	cfgBuf, err := json.Marshal(cfg)
+	if err != nil {
+		return false, fmt.Errorf("failed to marshal config: %v", err)
+	}
+
+	mergedCfgBuf, err := json.Marshal(mergedCfg)
+	if err != nil {
+		return false, fmt.Errorf("failed to marshal merged config: %v", err)
```

Comment:
Should we use `%w` here or is it the same?

Reply:
Don't feel the need to. The error is not a named one, and I don't foresee any scenarios inspecting the inner error.